falcon/request.py

```diff
@@ -1096,6 +1112,20 @@ def _parse_form_urlencoded(self):
 
             self._params.update(extra_params)
 
+    def _parse_application_json(self):
+        raw_body = self.stream.read()
+
+        body = None
+        try:
+            body = json.loads(raw_body.decode('utf-8'))
+        except UnicodeDecodeError:
```

Comment:
I don't think that JSON decoding errors should be ignored like that. Note also that `'null'` string is a valid JSON. So there is no way you can distinguish deserialisation error from valid `'null'` JSON value.